src/components/backend-ai-webui.ts

```diff
@@ -1430,13 +1458,16 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
                           <mwc-list-item class="horizontal layout start center" disabled style="border-bottom:1px solid #ccc;">
                               ${this.user_id}
                           </mwc-list-item>
+                          <mwc-list-item class="horizontal layout start center" disabled style="border-bottom:1px solid #ccc;">
+                              Role: ${this.roleInfo.role}
```

Comment:
How about marking the user's role with `assignment_ind` mwc-icon instead of Role text?
<img src="https://user-images.githubusercontent.com/28584164/125631202-c230ecba-4250-41d0-9b53-f43397b7d654.png" width="100"/>

Reply:
Which one does look more suited?

With icon:
![image](https://user-images.githubusercontent.com/49114533/125641402-d7c1269c-4d2b-4b54-b050-977f9670398e.png)
With text:
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/49114533/125641658-23da8529-3f4a-41cb-8755-bee3f5ad175e.png)

Reply:
In my opinion, this looks more consistent:
![image](https://user-images.githubusercontent.com/49114533/125645744-eb9b34fa-e5fb-4e4c-b203-6fd9f70c4d85.png)